### ISSUE TITLE
[Bug] `easymde` loading content only when clicking into it

### ISSUE THREAD

**Issue by BernhardK91** [Laravel-Backpack/CRUD#5353]
# Bug report

### What I did

I have an `easymde` field added to the CRUD controller as a subfield of a repeatable:

```php
[
    'name' => 'content',
    'label' => __('Content'),
    'type' => 'easymde',
],
```

I am using Tabler-Theme. The field is on and additional "pages" tab.

### What I expected to happen

When editing an existing entry, the content should be shown in the editor.

### What happened

When editing an existing entry, the content seams to be empty, until clicked into the editor. (In dark and lite mode).

It seems it works, when opening the tab directly `.../1/edit#pages`

![Animation1](https://github.com/Laravel-Backpack/CRUD/assets/5526334/a54827a5-eb81-4b93-8f05-114c33a943cb)

### What I've already tried to fix it

Nothing. 

### Is it a bug in the latest version of Backpack?

After I run ```composer update backpack/crud``` the bug... is it still there?

Yes.

### Backpack, Laravel, PHP, DB version

When I run ```php artisan backpack:version``` the output is:

```bash
### PHP VERSION:
PHP 8.2.10 (cli) (built: Aug 30 2023 09:46:43) (NTS Visual C++ 2019 x64)
Copyright (c) The PHP Group
Zend Engine v4.2.10, Copyright (c) Zend Technologies

### LARAVEL VERSION:
10.28.0.0

### BACKPACK PACKAGE VERSIONS:
backpack/basset: 1.2.1
backpack/crud: 6.2.4
backpack/generators: v4.0.2
backpack/pro: 2.0.16
backpack/theme-tabler: 1.1.1
```

**Comment by karandatwani92** [comment]
Hey @BernhardK91,

Thanks for putting this bug in notice. I have submitted a fix [here](https://github.com/Laravel-Backpack/PRO/pull/216).

Once it's tested! We will have it in the coming updates.

**Comment by pxpm** [comment]
Hey @BernhardK91 I've merged the fix in `backpack pro 2.0.20` just now. In a few moments our private repository will pick it and you should be able to do a `composer update backpack/pro` to get the latest version. 

Let me know if you are still experiencing the issue after the update and we can re-evaluate the issue. 

Cheers
